react/features/base/components/context-menu/ContextMenuItemGroup.tsx

```diff
@@ -1,24 +1,23 @@
-// @flow
 import { makeStyles } from '@material-ui/core';
-import React from 'react';
+import React, { ReactNode } from 'react';
 
-import ContextMenuItem from './ContextMenuItem';
+import ContextMenuItem, { Props as ItemProps } from './ContextMenuItem';
 
 
 type Props = {
 
     /**
      * List of actions in this group.
      */
-    actions?: Array<Object>,
+    actions?: Array<ItemProps>,
 
     /**
      * The children of the component.
      */
-    children?: React$Node,
+    children?: ReactNode,
 };
 
-const useStyles = makeStyles(theme => {
+const useStyles = makeStyles((theme: any) => {
```

Comment:
would it be complicated to create an Interface for theme? It would probably require some maintenance in the future as the theme is enhanced with new stuff though, but it would help with auto-complete when using the theme object.

Reply:
We already have one, but I'm not sure how it'll affect the upgrade process for material-ui, so I just used `any` for now. I'll change it after the upgrade